README.md

```diff
@@ -1,4 +1,4 @@
-# NI MEDIA [![Travis Build Status](https://travis-ci.org/NativeInstruments/ni-media.svg?branch=master)](https://travis-ci.org/NativeInstruments/ni-media/builds) [![Appveyor Build Status](https://ci.appveyor.com/api/projects/status/github/NativeInstruments/ni-media?branch=master&svg=true)](https://ci.appveyor.com/project/NativeInstruments/ni-media/history) [![codecov](https://codecov.io/gh/NativeInstruments/ni-media/branch/master/graph/badge.svg)](https://codecov.io/gh/NativeInstruments/ni-media)
+# NI MEDIA [![Build Status](https://dev.azure.com/mohinda/ni-media/_apis/build/status/marcrambo.ni-media?branchName=feature%2Fazure)](https://dev.azure.com/mohinda/ni-media/_build/latest?definitionId=1&branchName=feature%2Fazure-ci) [![codecov](https://codecov.io/gh/NativeInstruments/ni-media/branch/master/graph/badge.svg)](https://codecov.io/gh/NativeInstruments/ni-media)
```

Comment:
The ci is currently setup to my fork. As soon as the pr is merged i'll setup the pipeline for the ni repo and update the link.